src/Riter/Core/GlobalHotkeyManager.cs

```diff
@@ -82,7 +82,7 @@ public void Dispose()
     {
         foreach (var id in _hotkeyActions.Keys)
         {
-            UnregisterHotKey(_windowHandle, id);
+            UnregisterHotKey(_windowHandle, (int)id);
         }
 
         _source.RemoveHook(HwndHook);
```

Comment:
GC.SuppressFinalize(this) after disposing of resources to prevent the garbage collector from calling the finalizer.